web_widget_float_formula/static/src/js/web_widget_float_formula.js

```diff
@@ -85,12 +81,13 @@ odoo.define('web_widget_float_formula', function(require) {
             _compute_result: function() {
                 this._clean_formula_text();
 
-                var formula = this._process_formula(this.$el.find('input').val());
+                var formula = this._process_formula(this.$el.val());
                 if (formula !== false) {
                     var value = this._eval_formula(formula);
                     if (value !== false) {
                         this._formula_text = "=" + formula;
                         this.set_value(value);
+                        this.$el.val(value);
```

Comment:
This is circumventing the widget value set, which will not play nicely with inherited modules that may want to do other things with the value. Please remove - I'm almost certain nothing will break.